files/project-templates/webpack/spec/e2e/jasmine.json

```diff
@@ -0,0 +1,6 @@
+{
+    "spec_dir": "spec",
+    "spec_files": [
+        "e2e/**/*-spec.js"
+    ]
+}
```

Comment:
This is because I am going to release a new version of `webdriverio-server` that uses `jasmine` instead of `jasmine-node` which will look for this file when running `e2e` tests.